qemu/tests/rebase_second_snapshot_to_base.py

```diff
@@ -0,0 +1,108 @@
+import os
+import json
+import logging
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+from qemu.tests.qemu_disk_img import QemuImgTest
+from qemu.tests.qemu_disk_img import generate_base_snapshot_pair
+
+
+def run(test, params, env):
+    """
+    Rebase a second qcow2 snapshot to a raw base file.
+
+    1. create a qcow2 snapshot base -> sn1
+    2. boot the guest from the sn1
+    3. create a file in the snapshot disk,  calculate its md5sum
+    4. shut the guest down
+    5. create a qcow2 snapshot sn1 -> sn2
+    6. rebase the sn2 to the base
+    7. remove the sn1, optional
+    8. boot the guest from the sn2 and check whether the
+       file's md5sum stays same
+
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    def _get_img_obj_and_params(tag):
+        """Get an QemuImg object and its params based on the tag."""
+        img_param = params.object_params(tag)
+        img = QemuImg(img_param, data_dir.get_data_dir(), tag)
+        return img, img_param
+
+    def _get_compat_version():
+        """Get snapshot compat version."""
+        if params.get("image_extra_params") is None:
+            # default compat version for now is 1.1
+            return "1.1"
+        return params.get("image_extra_params").split("=")[1]
+
+    def _verify_qemu_img_info_backing_chain(output, b_fmt, b_name):
+        """Verify qemu-img info output for this case."""
+        logging.info("Verify snapshot's backing file information.")
+        res = json.loads(output)[:-1]
+        for idx, backing in enumerate(res):
+            if (backing["backing-filename-format"] != b_fmt[idx] or
+                    backing["backing-filename"] != b_name[idx]):
+                test.fail("Backing file information is not correct,"
+                          " got %s." % b_name[idx])
+            compat = backing["format-specific"]["data"]["compat"]
```

Comment:
This will cause an exception if we support it on RHEL-6. If the case doesn't support RHEL-6, it is better to add "no Host_RHEL.m6 in the confguration"

Reply:
updated, thanks